moduleframework/helpers/container_helper.py

```diff
@@ -128,35 +165,18 @@ def start(self, args="-it -d", command="/bin/bash"):
         :return: None
         """
         if not self.status():
-            if 'start' in self.info and self.info['start']:
+            if self.info.get('start'):
```

Comment:
What about traceback if 'start' does not exist?

Reply:
get returns None instead of traceback